grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -755,4 +789,199 @@ class OrderService {
 
         return isPending?:isApprover
     }
+
+    String exportOrderItems(List<OrderItem> orderItems) {
+        def rows = []
+        orderItems.each { orderItem ->
+            def row = [
+                    'Order Item ID' : orderItem?.id,
+                    'Pack level 1'  : '',
+                    'Pack level 2'  : '',
+                    Code            : orderItem?.product?.productCode,
+                    Product         : orderItem?.product?.name,
+                    UOM             : orderItem?.unitOfMeasure,
+                    Lot             : '',
+                    Expiry          : '',
```

Comment:
There should be format added in header (It's specified in ticket's description) so it should be "Expiry (dd/mm/yy)"